src/main/java/com/pushdown/automaton/controller/AutomataController.java

```diff
@@ -42,12 +42,9 @@ public AutomataController() {
 	@Qualifier("generadorAutomataPila")
 	private GeneradorAutomataPila generadorAutomataPila;
  
-	@WebEndpoint
-	@Timed
 	@RequestMapping(value = "/Generate", method = RequestMethod.POST)
 	public ResponseEntity<?> generaAutomata(@RequestBody String entrada, HttpServletRequest request) {
 		String entradaConsulta = Utils.correctorCharEspeciales(entrada);
-		logger.info(request.getLocalName() +" : " +request.getLocalAddr());

```

Comment:
umm... maybe remote name and address better ? not @borjarg95 ?

Reply:
fixed